package.json

```diff
@@ -50,11 +52,14 @@
     "pure-rand": "^1.6.2"
   },
   "devDependencies": {
```

Comment:
Please commit the package-lock.json instead of the yarn.lock. I prefer to only handle one locking system for the moment.